src/Handlers/Handlers.csproj

```diff
@@ -7,10 +7,12 @@
     <Nullable>enable</Nullable>
     <AssemblyName>WebForms.HttpHandler</AssemblyName>
     <RootNamespace>WebForms.HttpHandler</RootNamespace>
+    <WarningsAsErrors>Nullable</WarningsAsErrors>
```

Comment:
:)

Reply:
this is ok, right ? We will enforce across all projects before beta launch :)

Reply:
yup!